modules/combined/tests/elastic_thermal_patch/tests

```diff
@@ -1,33 +1,71 @@
 [Tests]
-  [./test]
+  [./elastic_thermal_patch]
     type = 'Exodiff'
     input = 'elastic_thermal_patch_test.i'
-    exodiff = 'out.e'
+    exodiff = 'elastic_thermal_patch_out.e'
     abs_zero = 1e-08
     use_old_floor = True
+    prereq = 'elastic_thermal_patch_sm'
   [../]
 
-  [./test_jac_rz_smp]
+  [./elastic_thermal_jacobian_rz_smp]
     type = 'Exodiff'
     input = 'elastic_thermal_jacobian_rz_smp_test.i'
-    exodiff = 'out_jac_rz_smp.e'
+    exodiff = 'elastic_thermal_jacobian_rz_smp_out.e'
     custom_cmp = 'elastic_thermal_patch.cmp'
     use_old_floor = True
     max_parallel = 1
+    prereq = 'elastic_thermal_jacobian_rz_smp_sm'
```

Comment:
for most of the tm tests, prereq is used to make both the sm and tm tests compare to the same gold file. But for the tests with two different gold files, i.e. one for sm and the other for tm, this is because the tm input gives more accurate results than sm input wrt analytical solution, and this results in exodiff failure for these tests if compared against the same sm gold file.